### PR TITLE
all pages responsive + sticky sidebar on scroll

### PR DESCRIPTION
**Pull request by Sarahb-da**
cleaned up responsiveness

### CODE REVIEW

File: ui/src/components/common/TopMenu.tsx [hunk 1]
```diff
@@ -42,6 +44,14 @@ const TopMenu: React.FC<Props> = ({ title, notifications, onLogout, topMenuButto
                             </Button>
                         </Menu.Item>
                     )}
+                    {topMenuButtons &&
+                        <Menu.Item className='overflow-menu-item'>
+                            <OverflowMenu>
+                                {topMenuButtons?.map(b =>
+                                    <OverflowMenuEntry label={b.label} onClick={b.onClick}/>
```
Comment (alexmatson-da) [inline ui/src/components/common/TopMenu.tsx:51]: this'll need a 🗝️  

(`b.label` should suffice, if it's same to assume each label is unique)

**Review by alexmatson-da — APPROVED** [review]
one comment, everything else looks good!

File: ui/src/components/common/TopMenu.tsx (hunk 1 above)
Comment (Sarahb-da) [reply ui/src/components/common/TopMenu.tsx:51]: oops, thanks!

**Review by alexgraham-da — APPROVED** [review]
looks good 👍 👍

**Pull request merged**